routes/verifier.php

```diff
@@ -0,0 +1,5 @@
+<?php
+if (getenv('APP_ENV') === 'local' || getenv('APP_ENV') === 'dev') {
```

Comment:
@rowasc @Angamanga 

in retrospective, I'm a bit on the fence about this APP_ENV check

As I was trying stuff out, when running the verification routines from platform-client, I was getting errors about these routes returning non-json replies. I had to go back to the backend code and look up these routes to understand what was going on. A good chunk of people may get stuck here.

Reply:
mmmmm . Yea, I see how that can happen. My only idea to make this simpler is to warn on those endpoints "the endpoint is disabled, check your APP_ENV" instead of not mounting them. ... which is sort of gross, but IDK...